.eslintrc

```diff
@@ -35,29 +37,57 @@
     "no-array-constructor": 2,
     "no-caller": 2,
     "no-catch-shadow": 0,
+    "no-cond-assign": 2,
+    "no-constant-condition": 2,
+    "no-control-regex": 2,
+    "no-debugger": 2,
+    "no-delete-var": 2,
+    "no-dupe-args": 2,
+    "no-dupe-keys": 2,
+    "no-duplicate-case": 2,
     "no-else-return": 2,
+    "no-empty": 2,
+    "no-empty-character-class": 2,
     "no-empty-label": 2,
     "no-eval": 2,
+    "no-ex-assign": 2,
     "no-extend-native": 2,
     "no-extra-bind": 2,
+    "no-extra-boolean-cast": 2,
     "no-extra-parens": [2, "functions"],
+    "no-extra-semi": 2,
+    "no-extra-strict": 2,
+    "no-fallthrough": 2,
+    "no-func-assign": 2,
     "no-implied-eval": 2,
+    "no-inner-declarations": [2, "functions"],
+    "no-invalid-regexp": 2,
+    "no-irregular-whitespace": 2,
     "no-iterator": 2,
     "no-label-var": 2,
     "no-labels": 2,
     "no-lone-blocks": 2,
     "no-loop-func": 2,
+    "no-mixed-requires": [2, false],
```

Comment:
The boolean config of this rule is deprecated according to the docs, this way is preferred:

    {"grouping": false}